charts/metrics/values.yaml

```diff
@@ -111,7 +111,7 @@ grafana-agent:
               wal_truncate_frequency: {{.Values.prom_config.wal_truncate_frequency}}
               remote_flush_deadline: {{.Values.prom_config.remote_flush_deadline}}
               remote_write:
-                - url: {{ include "observe.collectionEndpoint" . }}/v1/prometheus
+                - url: {{ include "observe.collectionEndpoint" . }}/v1/prometheus?clusterUid=${OBSERVE_CLUSTER}
```

Comment:
Similar comment to the one left in kustomize manifests around removing the `clusterUid` defined under `external_labels`. Once you have confirmation on that, you can update both places (helm and kustomize).

Reply:
Sure, got confirmation on that, and i have updated both.